pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -1699,7 +1698,7 @@ func readyIngressStatus() netv1alpha1.IngressStatus {
 	status.InitializeConditions()
 	status.MarkNetworkConfigured()
 	status.MarkLoadBalancerReady([]netv1alpha1.LoadBalancerIngressStatus{
-		{DomainInternal: resourcenames.K8sGatewayServiceFullname},
+		{DomainInternal: "knative-ingressgateway.istio-system.svc.cluster.local"},
```

Comment:
Should this use 

```go
reconciler.GetK8sServiceFullname("knative-ingressgateway", "istio-system")
```

for good measure?

Reply:
Looks better, but I think there's no essential difference unless we have way to avoid the hard code "knative-ingressgateway"/"istio-system".
I'll update anyway, as it looks cleaner.